fvm/transactionSequenceNum.go

```diff
@@ -94,7 +98,7 @@ func (c *TransactionSequenceNumberChecker) checkAndIncrementSequenceNumber(
 		txnState.RunWithAllLimitsDisabled(func() {
 			_ = txnState.RestartNestedTransaction(nestedTxnId)
 		})
-		return fmt.Errorf("checking sequence number failed: %w", err)
+		return err
```

Comment:
so we don't wrap `CodedError`?

Reply:
the wrapping is now done in `Process` (line 28).  Previously, we weren't consistently wrapping the coded error (line 68, 75, 97 were wrap correctly, but not line 82)